yarn.lock

```diff
@@ -2,13 +2,26 @@
 # yarn lockfile v1
 
 
+<<<<<<< Updated upstream
```

Comment:
yarn.lock is full of merge conflicts. Please roll back the changes in the lock file and let yarn update it automatically.